release-notes.txt

```diff
@@ -1,3 +1,232 @@
+RIOT-2015.12 - Release Notes
+============================
+RIOT is a real-time multi-threading operating system that supports a range of
+devices that are typically found in the Internet of Things: 8-bit
+microcontrollers, 16-bit microcontrollers and light-weight 32-bit processors.
+
+RIOT is based on the following design principles: energy-efficiency, real-time
+capabilities, small memory footprint, modularity, and uniform API access,
+independent of the underlying hardware (this API offers partial POSIX
+compliance).
+
+RIOT is developed by an international open source community which is
+independent of specific vendors (e.g. similarly to the Linux community).
+
+About this release:
+============
+This release is mostly a clean-up and bug-fixing release. Besides that, it introduces SAUL,
+the [S]ensor [A]ctuator [U]ber [L]ayer, which offers a unified API to interact with all
+different types of sensors and actuators on RIOT supported hardware. Furthermore, it re-enables
+the support for ICN by integrating CCN-Lite as a package and adds a lot of new overall
```

Comment:
I know I have written this sentence myself, but after re-reading it now, I think it would make sense to split it up into two sentences. One about the CCN-Lite thing, the other about documentation.